ip/event_manager.go

```diff
@@ -206,7 +210,11 @@ func (m *manager) healthChecker(ctx context.Context) {
 			m.sendHealthcheckResults(ctx, healthy, err)
 		}
 
-		time.Sleep(m.config.HealthcheckInterval)
+		interval := time.Duration(m.IP().HealthcheckInterval) * time.Second
```

Comment:
It's not a big computation, but don't you want to compute the interval outside of the loop?

Reply:
ok